src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -431,6 +437,27 @@ public void testCreateCourseWithWrongShortName() throws Exception {
         request.getMvc().perform(buildCreateCourse(course)).andExpect(status().isBadRequest());
     }
 
+    // Test
+    public void testCreateCourseWithDefaultChannels() throws Exception {
+
+        final List<String> defaultChannelNames = List.of("tech-support", "organization", "random", "announcement");
```

Comment:
These strings should be derived from the same list as the `AdminCourceResource` does, otherwise we will always have to adapt them twice.